src/vision.cs

```diff
@@ -804,40 +814,125 @@ interface VNSequenceRequestHandler {
 	[DisableDefaultCtor]
 	interface VNTargetedImageRequest {
 
-		[Export ("initWithTargetedCVPixelBuffer:")]
-		IntPtr Constructor (CVPixelBuffer pixelBuffer);
+		[Export ("initWithTargetedCVPixelBuffer:options:")]
+		IntPtr Constructor (CVPixelBuffer pixelBuffer, NSDictionary optionsDict);
```

Comment:
Unless it's confusing use the same parameters name as Apple, i.e. the `Dict` suffix is not needed, `options` is fine.